src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -58,176 +58,234 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
+                            // Version match found, the package status is defined by the vulnerability status.
                             if (version->status() == NSVulnerabilityScanner::Status::Status_affected)
                             {
-                                logDebug1(WM_VULNSCAN_LOGTAG,
-                                          "Match found, the package '%s', is vulnerable to '%s'. Current version: "
-                                          "'%s' ("
-                                          "less than '%s' or equal to '%s'). - Agent '%s' (ID: '%s', Version: '%s').",
-                                          packageName.c_str(),
-                                          callbackData.cveId()->str().c_str(),
-                                          packageVersion.c_str(),
-                                          versionStringLessThan.c_str(),
-                                          versionStringLessThanOrEqual.c_str(),
-                                          data->agentName().data(),
-                                          data->agentId().data(),
-                                          data->agentVersion().data());
+                                logDebug1(
+                                    WM_VULNSCAN_LOGTAG,
+                                    "Match found, the package '%s', is vulnerable to '%s'. Current version: '%s' is "
+                                    "equal to '%s'. - Agent '%s' (ID: '%s', Version: '%s').",
+                                    packageName.c_str(),
+                                    callbackData.cveId()->str().c_str(),
+                                    packageVersion.c_str(),
+                                    versionString.c_str(),
+                                    data->agentName().data(),
+                                    data->agentId().data(),
+                                    data->agentVersion().data());
 
                                 data->m_elements[callbackData.cveId()->str()] = nlohmann::json::object();
                                 data->m_alerts[callbackData.cveId()->str()]["vulnerability"]["package"]["condition"] =
-                                    versionStringLessThan.empty()
-                                        ? "Package less than or equal to " + versionStringLessThanOrEqual
-                                        : "Package less than " + versionStringLessThan;
+                                    "Package equal to " + versionString;
 
                                 return true;
                             }
+
+                            return false;
+                        }
+                    }
+                    else
+                    {
+                        // Version range specified
+
+                        // Check if the installed version satisfies the lower bound of the version range.
+                        auto lowerBoundMatch = false;
+                        if (versionString.compare("0") == 0)
```

Comment:
This is not a valid version on any version matcher, we have to separate this check (due to the exception on the version matcher)